config/locales/en.yml

```diff
@@ -223,7 +223,7 @@ en:
   index:
    title: 'Medications'
    subtitle: 'Keep track of your medications, including dosage and refill information.'
-   total: 'Total:'
+   total: 'Total'
```

Comment:
I don't think the quotes here are necessary, AFAIK in locale files quotes are only necessary for interpolation